README.md

```diff
@@ -2,4 +2,32 @@
 
 The aim of this project is to build a roman number converter that will help further my understanding of coding in a functional programming language in this case elixir.
 
```

Comment:
Actually, there is one thing - the readme should have an instruction on how to run in and how to test it.